polaris-react/src/components/Checkbox/Checkbox.scss

```diff
@@ -62,6 +65,19 @@
   height: 100%;
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
   @include focus-ring($border-width: var(--p-border-width-2));
+
+  #{$se23} & {
+    border: var(--p-border-width-1-experimental) solid
+      var(--p-color-border-input);
+  }
+}
+
+.Backdrop:hover,
+.Backdrop.hover {
+  #{$se23} & {
+    border: var(--p-border-width-2-experimental) solid
+      var(--p-color-border-input-hover);
+  }
 }
```

Comment:
I was shocked there isn't hover styles somewhere already. Should we move this close to other backdrop class names? Unsure if it was here for specificity.